csharp/ql/src/semmle/code/csharp/frameworks/microsoft/AspNetCore.qll

```diff
@@ -298,6 +298,44 @@ class MicrosoftAspNetCoreHttpHeaderDictionaryExtensions extends RefType {
   Method getSetCommaSeparatedValuesMethod() { result = this.getAMethod("SetCommaSeparatedValues") }
 }
 
+/** The `Microsoft.AspNetCore.Http.CookieOptions` class. */
+class MicrosoftAspNetCoreHttpCookieOptions extends RefType {
+  MicrosoftAspNetCoreHttpCookieOptions() {
+    this.hasQualifiedName("Microsoft.AspNetCore.Http", "CookieOptions")
+  }
+}
+
+/** The `Microsoft.AspNetCore.Http.CookieBuilder` class. */
+class MicrosoftAspNetCoreHttpCookieBuilder extends RefType {
+  MicrosoftAspNetCoreHttpCookieBuilder() {
+    this.hasQualifiedName("Microsoft.AspNetCore.Http", "CookieBuilder")
+  }
+}
+
+/** The `Microsoft.AspNetCore.Builder.CookiePolicyOptions` class. */
+class MicrosoftAspNetCoreBuilderCookiePolicyOptions extends RefType {
+  MicrosoftAspNetCoreBuilderCookiePolicyOptions() {
+    this.hasQualifiedName("Microsoft.AspNetCore.Builder", "CookiePolicyOptions")
+  }
+}
+
+/** The `Microsoft.AspNetCore.Authentication.Cookies.CookieAuthenticationOptions` class. */
+class MicrosoftAspNetCoreAuthenticationCookiesCookieAuthenticationOptions extends RefType {
+  MicrosoftAspNetCoreAuthenticationCookiesCookieAuthenticationOptions() {
+    this.hasQualifiedName("Microsoft.AspNetCore.Authentication.Cookies", "CookieAuthenticationOptions")
```

Comment:
Some of our tests are complaining that this file was not autoformatted. I think this is the violating line, as it's too long.